dev-docs/bidders/rtbdemand_com.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: bidder
+title: RtbDemand.com
+description: Prebid RtbDemand.com Bidder Adaptor
+pbjs: true
+pbs: true
```

Comment:
I don't see a PR for a server-side adapter.

Reply:
I guess PR for PBS isn't required for aliased adaptor

Reply:
@bretg metadata has been updated for all related aliases